src/screens/Settings/components/GeneralSettings/index.tsx

```diff
@@ -246,6 +252,23 @@ export default function GeneralSettings({
       </span>
       <span className="setting">
         <span className="toggleWrapper">
+          {t('setting.adddesktopshortcuts', 'Add desktop shortcuts automatically')} (Linux)
```

Comment:
Maybe it's better to add a platform check here. We have a platform from contextProvider, so just check if is Linux and render those options conditionally.

Reply:
Check line 72 for an example

Reply:
Ok

Reply:
What if I disable the field if platform != linux? and keep the (Linux) parenthesis.?

Reply:
But why? Since it will appear only for Linux?

Reply:
What about cheking (!isWindows) as we do for other options instead of implementing now (Linux). Makes the code more readable.

Reply:
> But why? Since it will appear only for Linux?

This way they'll know that the feature exists and they'll not ask "hey, add x"

Reply:
> What about cheking (!isWindows) as we do for other options instead of implementing now (Linux). Makes the code more readable.

This will make mac support even harder.

Reply:
> > But why? Since it will appear only for Linux?
> 
> This way they'll know that the feature exists and they'll not ask "hey, add x"

So let it be the way it is, for all platforms and before next release I'll make it work for Windows and mac.
Don't add the Linux in parenthesis and also let is show for all platforms.
After you merge this I'll check how we can do on mac and Windows. 🙂